### PR TITLE
[0.4.x] libvisual-plugins: Update actor_pixbuf to build with GTK 3.x

### PR DESCRIPTION
**Pull request by hartwork**
0.4.x backport of pull request #177

### CODE REVIEW

File: libvisual-plugins/configure.ac [hunk 1]
```diff
@@ -119,15 +119,15 @@ AC_ARG_ENABLE([gdkpixbuf-plugin],
...
 if test "$ENABLE_PLUGIN_GDKPIXBUF" = "yes"; then
-  PKG_CHECK_MODULES([GTK], [gtk+-2.0 >= gtk_required_version],
+  PKG_CHECK_MODULES([GTK], [gtk+-3.0 >= gtk_required_version],
     [HAVE_GTK="yes"], [HAVE_GTK="no"])
 
   if test "$HAVE_GTK" = "yes"; then
     build_actor_plugins="$build_actor_plugins gdkpixbuf"
   else
     AC_MSG_WARN([*** GTK+ >= ${gtk_required_version} is not found.
 	  	The libvisual GdkPixbuf image loader plugin won't be build.
```
Comment (kaixiong) [inline libvisual-plugins/configure.ac:129]: Correct 'build' to 'built'.

File: libvisual-plugins/configure.ac [hunk 2]
```diff
@@ -119,15 +119,15 @@ AC_ARG_ENABLE([gdkpixbuf-plugin],
...
-		GdkPixbuf is included within gtk-2.0 and newer, which can be
+		GdkPixbuf is included within gtk+-3.0 and newer, which can be
 		downloaded at http://www.gtk.org/])
```
Comment (kaixiong) [inline libvisual-plugins/configure.ac:131]: Correct URL to `https://www.gtk.org`.

**Review by kaixiong — CHANGES_REQUESTED** [review]
Looks good to go, just 2 minor updates to add.

**Comment by hartwork** [comment]
> Looks good to go, just 2 minor updates to add.

@kaixiong I was unsure whether to make these adjustments one more commit, two more commits, or amend into a single commit. I went for the last option for now, but I can change it as needed. Pushed.

**Review by kaixiong — APPROVED** [review]
@hartwork, I was okay with either way. Let's close this :)

**Pull request merged**